example/helm/values.mysql.yaml

```diff
@@ -4,3 +4,25 @@ auth:
   existingSecret: db-credentials
 commonLabels:
   sidecar.istio.io/inject: "false"
+
+initdbScripts:
```

Comment:
I also not sure if this is right approach to tightly such scripts with database deployment itself

Bitnami mentions it's only for fresh instance and first boot, should we consider a separate Job for database initialisation instead ?

Reply:
This is for example configuration. Having this in mind, it should be fine. Our main product is the Helm Chart and the example configuration is just for reference. But, we should document this.

I created an issue for documentation here: https://github.com/kromanow94/kubeflow-manifests/issues/10